### PR TITLE
SLEM: Extra repo for software packages

### PR DESCRIPTION
**Pull request by tbazant**
### Description

A new software pool will be available to customers for testing purposes

### Are there any relevant issues/feature requests?

* jsc#PED-7901

### Result

In the end, I did more changes on the tr-up front:
* added info about software pools
* created few topics from sections that deserved it
* lifted several sections up and flattened the article
* replaced root prompt with sudo prompt
* XML-formatted where the source looked messy

### CODE REVIEW

File: articles/transactional-updates.asm.xml [hunk 1]
```diff
@@ -55,107 +59,111 @@
...
-        <varlistentry>
-          <term>GOAL</term>
+          </varlistentry>
+          <varlistentry>
+            <term>GOAL</term>
             <listitem>
               <para>
-                You will understand how the &tr-up; works and how you can use it to administer your system.
+                You will understand how the &tr-up; works and how you can use it
+                to administer your system.
```
Comment (dariavladykina) [inline articles/transactional-updates.asm.xml:161]:
```suggestion
                You will understand how &tr-up; works and how you can use it
                to administer your system.
```

File: articles/transactional-updates.asm.xml [hunk 2]
```diff
@@ -55,107 +59,111 @@
...
+                and do not influence the running system until you boot into the
+                snapshot.
+              </para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>WHY?</term>
+            <listitem>
+              <para>
+                You want to administer &productnameshort;.
```
Comment (dariavladykina) [inline articles/transactional-updates.asm.xml:144]: Suggest rephrase:

You want to administer &productnameshort; and manage its updates, minimizing the risk of system downtime due to update failures while ensuring easy rollbacks.

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some comments here. Thanks!

File: articles/transactional-updates.asm.xml (hunk 2 above)
Comment (tbazant) [inline articles/transactional-updates.asm.xml:144]:
```suggestion
                You want to administer &productnameshort; and manage its updates, minimizing the risk of system downtime due to update failures while ensuring easy rollbacks.
```

File: tasks/transactional-update-cleanup.xml [hunk 3]
```diff
@@ -16,16 +16,52 @@
  xmlns:xlink="http://www.w3.org/1999/xlink"
  xmlns:trans="http://docbook.org/ns/transclusion">
   <info>
-    <title>Performing snapshots cleanup</title><!-- can be changed via merge
+    <title>Performing snapshots cleanup</title>
```
Comment (tbazant) [inline tasks/transactional-update-cleanup.xml:19]:
```suggestion
    <title>Performing snapshots cleanup using &tr-up;</title>
```

File: tasks/transactional-update-cleanup.xml [hunk 4]
```diff
@@ -16,16 +16,52 @@
...
+<!-- add author's e-mail -->
     <meta name="maintainer" content="jsindelarova@suse.com" its:translate="no"/>
-    <abstract><!-- can be changed via merge in the assembly -->
+    <abstract>
       <para>
-       The topic describes how the snapshot cleanup algorithm works.
+        You can use &tr-up; to clean unused file system snapshots and
+        unreferenced <filename>/etc</filename> overlay directories.
       </para>
     </abstract>
```
[the remark this answers is not in the document]
Comment (tbazant) [reply tasks/transactional-update-cleanup.xml:29]: it is again...

**Review by tbazant — COMMENTED** [review]
applied suggestions, suggesting few new ones as well

**Pull request merged**